AppCenter/AppCenter/Internals/Sender/MSSender.h

```diff
@@ -25,9 +25,10 @@ NS_ASSUME_NONNULL_BEGIN
  * Send data.
  *
  * @param data Instance that will be transformed to request body.
+ * @param appSecret The app secret.
  * @param handler Completion handler.
  */
-- (void)sendAsync:(nullable NSObject *)data completionHandler:(MSSendAsyncCompletionHandler)handler;
+- (void)sendAsync:(nullable NSObject *)data appSecret:(nullable NSString *)appSecret completionHandler:(MSSendAsyncCompletionHandler)handler;
```

Comment:
`appSecret` is always nullable. Why is it only specified in Sender?

Reply:
It should just not be nullable on the setAppSecret maybe.